src/docs/self-hosted/troubleshooting.mdx

```diff
@@ -49,6 +48,25 @@ The _proper_ solution is as follows ([reported](https://github.com/getsentry/onp
 You can replace <code>snuba-consumers</code> with other consumer groups or <code>events</code> with other topics when needed.
 </Alert>
 
+#### Nuclear option
+The "nuclear option" is removing all Kafka-related volumes and recreating them which _will_ cause data loss. Any data that was pending there will be gone upon deleting these volumes.
+1. Stop instance:
+   ```shell
+   docker-compose stop
```

Comment:
This will not work 100% of the time as `stop` keeps the containers around. That means everyone will face the same issue as you did. Let's give them the correct solution right off the bat.

```suggestion
   docker-compose down --volumes
```